src/js/tracks/text-track.js

```diff
@@ -339,14 +344,17 @@ class TextTrack extends Track {
   addCue(originalCue) {
     let cue = originalCue;
 
-    if (!(originalCue instanceof window.vttjs.VTTCue)) {
+    if (window.vttjs && !(originalCue instanceof window.vttjs.VTTCue)) {
       cue = new window.vttjs.VTTCue(originalCue.startTime, originalCue.endTime, originalCue.text);
 
       for (const prop in originalCue) {
         if (!(prop in cue)) {
```

Comment:
should we just do `|| prop === id` instead of copying it below? Doesn't really matter all that much I guess though.